assert/assertions_test.go

```diff
@@ -1,7 +1,8 @@
 package assert
 
 import (
-	"bytes"
+    "bufio"
```

Comment:
The indentation on these 2 new additions looks incorrect, could you please line them up with the rest of the imports? We use 4-character wide _tabs_